tools/RAiDER/constants.py

```diff
@@ -18,3 +18,6 @@
 
 R_EARTH_MAX = 6378137
 R_EARTH_MIN = 6356752
+
+_THRESHOLD_SECONDS = 5 * 60 # Threshold delta_time in seconds
```

Comment:
see my comment below

Reply:
@dbekaert Note that this particular threshold is for whether or not to interpolate. If the threshold is close enough, the code will just take the closest date. Threshold could be anything, I just picked 5 minutes and set it as a constant so it can be changed if desired.